src/plugins/intel_cpu/src/nodes/mha.cpp

```diff
@@ -1026,7 +1026,7 @@ void MHA::prepareParams() {
 
                 // don't create brgemm kernels for empty tiles
                 if (M_ != 0 && K_ != 0 && N_ != 0) {
-                    if (brg1BaseIdx == static_cast<size_t>(-1))
+                    if (brg1BaseIdx == std::numeric_limits<size_t>::max())
```

Comment:
Plaese pay attention to line 1005, we have to change the `brg1BaseIdx ` initialization with `std::numeric_limits<size_t>::max()` to preserve correctness.

Reply:
Done